### ISSUE TITLE
Accessibility check

### ISSUE THREAD

**Issue by csbailey5t** [dickensnotes/ddnp#20]
- check contrast of text and background colors

**Comment by dnoneill** [comment]
https://wave.webaim.org/extension/

**Comment by annamgibson** [comment]
Adam and I have gone through and shortened some ALT text per the accessibility check. Most of the issues coming up are about skipped headings or other small issues that I don't think seem to be big issues. There are a few problems with missing alt text in areas I don't know how to fix. 

- [ ] Homepage: Alerts, missing first-level heading on Hero image, and skipped heading level on the Bleak House card
- [ ] General Introduction: Alert: Redundant Alt text for Hard Times image (not sure this is a big issue; I don't see a way to easily edit the alt text or the caption, since they're short and just define the image). 
- [ ] Teaching with the DDNP: Alert: Skipped heading level (before "Instructions"); Suspicious link text (to the PDF); Link to PDF document
- [ ] BOTH Bleak House AND David Copperfield Working Notes: ERROR: Missing alt text on card image. (also an Alert: Possible heading) <img width="1263" alt="image" src="https://user-images.githubusercontent.com/84870158/206076470-3a9debe9-c6bb-4513-8def-c93e7b593b5c.png">
- [ ] The Team: ERRORS: It's claiming that the profile pics don't have alt text, but it looks to me like they do! Not sure why it says this. <img width="866" alt="image" src="https://user-images.githubusercontent.com/84870158/206077216-058b8e95-eb8e-44f0-949f-e2e28fc777e7.png">  <img width="412" alt="image" src="https://user-images.githubusercontent.com/84870158/206077252-36de9324-f2fa-4d48-9051-29c2774394cc.png">
- [ ] Editing the Notes: skipped heading level (Ink Colors)
- [ ] Bibliography: skipped heading level (before subhead for Bibliography after intro text)

**Comment by csbailey5t** [comment]
I've fixed most of these. Not sure why the one piece is being tagged as a suspicious link.
